docs/docsite/rst/plugins/connection.rst

```diff
@@ -9,7 +9,7 @@ Connection plugins
 
 Connection plugins allow Ansible to connect to the target hosts so it can execute tasks on them. Ansible ships with many connection plugins, but only one can be used per host at a time.
 
-By default, Ansible ships with several connection plugins. The most commonly used are the :ref:`paramiko SSH<paramiko_connection>`, native ssh (just called :ref:`ssh<ssh_connection>`), and :ref:`local<local_connection>` connection types.  All of these can be used in playbooks and with :command:`/usr/bin/ansible` to decide how you want to talk to remote machines. If necessary, you can :ref:`create custom connection plugins <developing_connection_plugins>`.
+By default, Ansible ships with several connection plugins. The most commonly used are the :ref:`paramiko SSH<paramiko_connection>`, native ssh (just called :ref:`ssh<ssh_connection>`), and :ref:`local<local_connection>` connection types.  All of these can be used in playbooks and with :command:`/usr/bin/ansible` to decide how you want to talk to remote machines. If necessary, you can :ref:`create custom connection plugins <developing_connection_plugins>`. You can use the ``ANSIBLE_CONNECTION_PATH`` environment variable to specify a custom directory for the ``ansible-connection`` script along with other files related to connections during remote execution. To change the connection plugin for your tasks, you can use the ``connection`` keyword.  
```

Comment:
I hate to keep coming back with changes but, looking at this with fresh eyes, I wonder if the description I suggested earlier is too verbose and possibly misleading.

Also @s-hertel mentioned `DEFAULT_CONNECTION_PLUGIN_PATH` in an previous comment: https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-connection-plugin-path

@s-hertel Should `ANSIBLE_CONNECTION_PATH` be replaced with `DEFAULT_CONNECTION_PLUGIN_PATH`?

Perhaps we could simplify this update with a much briefer mention of `DEFAULT_CONNECTION_PLUGIN_PATH` and a pointer to that section of the docs.

Reply:
@oraNod Yes, your assessment here is spot on https://github.com/ansible/ansible-documentation/pull/1296#discussion_r1578203817. This config is for something else (and actually in the process of being removed because the `ansible-connection` is getting an update).

But as I said, I really do not think we should be mentioning the correct config, and definitely not as a one-off on a certain type of plugin. It's a legacy feature, and most of the user-docs just recommend FQCN (which really defeats the purpose of a legacy-style override for something in ansible-core). The section below discussing `connection_plugins` directory is also a legacy feature and should be deleted from this page imo.

The developer docs are much more complete on this topic, and already linked. I don't think these docs should duplicate 'how to use legacy plugins' here. Most people just use collections so they should not configure a collection plugins path, they just need to use the FQCN.

Reply:
```suggestion
By default, Ansible ships with several connection plugins. The most commonly used are the :ref:`paramiko SSH<paramiko_connection>`, native ssh (just called :ref:`ssh<ssh_connection>`), and :ref:`local<local_connection>` connection types.  All of these can be used in playbooks and with :command:`/usr/bin/ansible` to decide how you want to talk to remote machines. If necessary, you can :ref:`create custom connection plugins <developing_connection_plugins>`. To change the connection plugin for your tasks, you can use the ``connection`` keyword.  
```

This is about to have no effect and be deprecated https://github.com/ansible/ansible/pull/82992/files#diff-f2a6cd50a2b6a7cda25339138821249850db0ef404138ebfcb4d4e7e90ae0d55.